manifests/params.pp

```diff
@@ -8,7 +8,7 @@
   # parameters dealing with installation
   $install_server = true
   $install_client = true
-  $release        = '3.12'
+  $release        = '7'
```

Comment:
https://www.gluster.org/install/ states version 6 is the latest released version.

Reply:
fixed it